### PR TITLE
[Docs page] Add footer links 

### PR DESCRIPTION
**Pull request by faxu**
Adds link to file doc issues and direct link to edit documentation on gh-pages branch
Staged at: https://faxu.github.io/onnxruntime/docs/

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -25,3 +25,12 @@ aux_links:
...
 ga_tracking: UA-156955408-1    
+
+# Footer "Edit this page on GitHub" link text
+gh_edit_link: true # show or hide edit this page link
+gh_edit_link_text: "Edit this page on GitHub"
+gh_edit_repository: "https://github.com/microsoft/onnxruntime" # the github URL for your repo
+gh_edit_branch: "gh-pages" # the branch that your docs is served from
+# gh_edit_source: docs # the source that your files originate from
+gh_edit_view_mode: "tree" # "tree" or "edit" if you want the user to jump into the editor immediately
+footer_content: "For documentation questions, please <a href=\"https://github.com/microsoft/onnxruntime/issues/new?labels=component:documentation&title=[Documentation%20Request]\" target=\"_blank\">file an issue</a>."
```
Comment (lenisha) [inline _config.yml:36]: Looks like **footer_content** is deprecated https://github.com/pmarsceill/just-the-docs/blob/master/docs/configuration.md#footer-content

**Review by lenisha — COMMENTED** [review]
Would be also great to add link to the actual page in the issue that is being opened by adding **body** parameter to the new issue link, like in ms docs example they format the body markup:

https://github.com/MicrosoftDocs/terminal/issues/new?title=&body=%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20docs.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%20d3cbeb9a-9e7c-f8f4-5c03-5d6262d2b440%0A*%20Version%20Independent%20ID%3A%20eb575c79-5b2a-45f6-646f-9d41d5d223e8%0A*%20Content%3A%20%5BWindows%20Terminal%20Powerline%20Setup%5D(https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fwindows%2Fterminal%2Ftutorials%2Fpowerline-setup)%0A*%20Content%20Source%3A%20%5BTerminalDocs%2Ftutorials%2Fpowerline-setup.md%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fterminal%2Fblob%2Fmaster%2FTerminalDocs%2Ftutorials%2Fpowerline-setup.md)%0A*%20Product%3A%20**dev-environment**%0A*%20Technology%3A%20**windows-terminal**%0A*%20GitHub%20Login%3A%20%40cinnamon-msft%0A*%20Microsoft%20Alias%3A%20**cinnamon**

File: _config.yml (hunk 1 above)
Comment (natke) [reply _config.yml:36]: We can add a footer.html template: https://jekyllrb.com/docs/includes/

Comment (faxu) [reply _config.yml]: Updated to use the _includes directory.

Comment (lenisha) [reply _config.yml]: change footer content to get the body populated with current page info:


{%- assign url = page.url -%}

<p class="text-small text-grey-dk-100 mb-0">For documentation questions, please <a href="https://github.com/microsoft/onnxruntime/issues/new?labels=component:documentation&title=[Documentation%20Request]&body=%5BEnter%20feedback%20here%5D%0A%0A---%0A%23%23%23%23%20Document%20Details%0A*%20Title%3A%20{{ page.title | url_encode }}%0A*%20Page%3A%20%5B{{ url | absolute_url | url_encode }}%5D({{ url | absolute_url | url_encode }})%0A*%20Page%20Source%3A%20{{ page.name | url_encode }}" target="_blank">file an issue</a></p>
© 2021 GitHub, Inc.

**Pull request closed without merging**